as/plugins/org.jboss.ide.eclipse.as.wtp.core/src/org/jboss/ide/eclipse/as/core/server/UnitedServerListenerManager.java

```diff
@@ -48,6 +48,14 @@ public static synchronized UnitedServerListenerManager getDefault() {
 	protected ArrayList<UnitedServerListener> list;
 	private UnitedServerListenerManager() {
 		list = new ArrayList<UnitedServerListener>();
+		new Thread() {

```

Comment:
what is the reason for threading here? Afaik WTP is threaded so even without this thread here you'd need to syncronize, right?